buildinfo/buildinfo.go

```diff
@@ -73,6 +75,16 @@ func IsDev() bool {
 	return strings.HasPrefix(Version(), develPrefix)
 }
 
+// IsSlim returns true if this is a slim build.
+func IsSlim() bool {
+	return strings.Contains(slim, "t")
```

Comment:
Could we set this via build tag so that it's present when doing `go build` as well?

Reply:
Like by using `go build -tags slim`?

Reply:
Yup

Reply:
Sure, but we won't be able to detect agpl this way since we don't use a flag for it.

Reply:
Done for slim, AGPL still uses ldflags. I've added a comment to `build_go.sh` why we don't provide an `agpl` build flag